### ISSUE TITLE
text size is too small 

### ISSUE THREAD

**Issue by changchichung** [lupoDharkael/flameshot#327]
thanks for the wonderful software, it`s just fantastic.
I just upgrade to 0.6.0 for the text tool, it works like a charm, but the font size is just too small, here is the screenshot. But I found nothing to configure about text in settings. is there anyway to change the default font size ?

![https://i.imgur.com/fHw2C2x.png](https://i.imgur.com/fHw2C2x.png)

**Comment by farseerfc** [comment]
Try use your mouse wheel to change the font size. And you can change the font settings by pressing space to bring out the control panel.

**Comment by orschiro** [comment]
> Try use your mouse wheel to change the font size. And you can change the font settings by pressing space to bring out the control panel.

Great! How come this is such a hidden feature? 

Can we make this more obvious? :-)

**Comment by bksubhuti** [comment]
I had to Google to find out how to increase the font.  Not good although this is the most awesome screen taker app!!!  How about a mouse selector tool that can select the text and stretch it?  This is the  obvious method that I looked for.

**Comment by JesseChavez** [comment]
The best screenshot by far, but this is a super hidden feature

**Comment by prosoitos** [comment]
Does this feature still exist? The mouse wheel doesn't do anything on my system and there is no font or size option in the control panel. Thanks!

OS: 5.9.14-arch1-1
flameshot: 0.8.5-1

**Comment by borgmanJeremy** [comment]
@prosoitos Yes you should be able to use the scroll wheel after selecting the tool to increase or decease size. You may also hit "spacebar" or click the purple icon on the left side of the screen to bring up the control panel. However you must do this AFTER clicking the text tool.

Let me know if you still have issues.

**Comment by prosoitos** [comment]
Oh, I see. I misunderstood what this issue was about then. My bad.

My problem is about increasing the size of the UI. On my 4k monitor, it is absolutely minuscule. Thank you! (Let me know if you would like me to open another issue since this is an unrelated question).

**Comment by cheapy** [comment]
If all computers had scroll wheels this would be ideal.  I wish my laptops had them.  Suggest simple Ctrl+/Ctrl- or Up/Down keys to change font size in text mode, or even a default font size setting in the ini file.  I had to use the spacebar.  This is tricky (until you find this screen), but I also like the program a lot, especially now that I know how to adjust the text.

**Comment by mmahmoudian** [comment]
@cheapy this has been investigated and even implemented by @Martin-Eckleben so that people with tablets be able (who doesn't even have Ctrl key) can use Flameshot smoothly. I'm not sure in what stage the merging is though. Perhaps @borgmanJeremy knows better.

**Comment by cheapy** [comment]
Thanks, I will watch for an update.  I am running antiX, which is Debian based, but many times will update earlier.

**Comment by mmahmoudian** [comment]
@cheapy that feature has not yet been merged into the main branch, but I would suggest that you follow that Pull Request (#1131) for when it is merged. You can also join the Slack to get more frequent updates and discussions in the dev channel:

https://join.slack.com/t/flameshotworkspace/shared_invite/zt-kx50keyc-H2GHTjQmmC4X2oRgSeDIVg

**Comment by cheapy** [comment]
I looked at the code and found a simple workaround for me.  I just changed the base point size from 8 to 14 in texttool.cpp and rebuilt it. 
`// SPDX-License-Identifier: GPL-3.0-or-later
// SPDX-FileCopyrightText: 2017-2019 Alejandro Sirgo Rica & Contributors

#include "texttool.h"
#include "textconfig.h"
#include "textwidget.h"

#define BASE_POINT_SIZE 14
`

**Comment by mmahmoudian** [comment]
@cheapy although it is mot a universal fix, but I'm glad that you are utilising the OpenSource aspect of the project 😊

**Comment by cheapy** [comment]
I apologize.  I could see that the code referenced above hasn't been implemented for a while.  I couldn't figure out how to install, compile or test those changes. 

And if I knew C++, I would have tried to put the default value in the .ini file instead of it being hard coded.  

But since it was hard coded, and I was able to compile and test it, I decided at least this would solve the problem for me.  Maybe a default value of 11 or 12 mould be better for many people.

**Comment by mmahmoudian** [comment]
@cheapy There is of course an alternative way: by pressing <kbd>space bar</kbd> or by clicking on the button on the left hand side of your monitor you can open the "Tool Settings" menu and from the top slider you can configure the thickness.

**Comment by boogiefromzk** [comment]
It's very hard to change font size with touchpad scroll, because it stops and then jumps unpredictable.
It would be amazing if font size would be in settings.
The rest is awesome!

**Comment by mmahmoudian** [comment]
@boogiefromzk considering that the font size and thickness are very frequently changed, it doesn't make any sense to have it in the config file especially when people version control their config files.

That said, in a recent version of Flameshot we have added two new buttons to assist people with touchpads and touch screens. Right click on the tray icon, in the first tab of configuration, scroll down and enable the <kbd>+</kbd> and <kbd>-</kbd>. This will add teo new buttons to the UI to change the size and thickness with.

As [I said just before your comment](https://github.com/flameshot-org/flameshot/issues/327#issuecomment-787794598) you can press spacebar and change the color and size from the side menu.

**Comment by boogiefromzk** [comment]
Wow, that's unbelivable fast answer, thank you, @mmahmoudian!
I hope I'll have those buttons in a few in my OpenSUSE, but here is what I've found for ones, who are dumb like me :) there is a way to change size of tools, while they are selected:
- Click on Text tool
- Add some text
- Click on Tool settings
- Change Active thickness

It also changes other tools size unfortunately, but still useful.

![image](https://user-images.githubusercontent.com/8693010/127809922-feb1d8b1-6cac-462f-9394-155f98b2ed43.png)

**Comment by boogiefromzk** [comment]
@mmahmoudian by the way, if I have default font size as I need, I wouldn't change it most of the time. I love flameshot for that I don't need to change anything after making default setup. But the font is not remembered. 
Now I need to adjust thickness and then font size and then arrow thickness and then back to font. It takes too much time.

See how Active thickness correspond to font size:
![image](https://user-images.githubusercontent.com/8693010/127810761-27ac6b6e-1ec4-472d-b90d-618bf2bde688.png)

**Comment by quicksketch** [comment]
Thanks @mmahmoudian for the improvements (and space bar trick)! I agree with @boogiefromzk that the font-size to line thickness ratio is too much, where the lines are disproportionately thick. I have to adjust the font size on every screenshot I take.

But I think that may be subjective as to what the right ratio is. What I would love to see as an alternative is to separate font-size from thickness. Each tool would keep a separate setting and changing one would not affect the other.

**Comment by mmahmoudian** [comment]
@boogiefromzk and @quicksketch the issue you raised about the "Ratio between text size and tool size" has been reported in #665 which was somehow addressed in #782 and it is already available in our latest version (v0.10.1). You can see what is your settings in your config file:

![image](https://user-images.githubusercontent.com/390889/131546732-387edc15-1b7d-4beb-8895-780814002df7.png)
 

But to be fair, the implemented solution is just to separate the text size and thickness and they are completely independent (no ratio). If you would like to have them as ratio, I suggest opening a new issue for the feature request, but before that perhaps it would be nice to also suggest how it is implemented. I mean how should it be denoted in the config file.

**Comment by boogiefromzk** [comment]
Thank you! For me on Linux there is no `drawFontSize` in the file after installation of version 0.10.1, but after I added it to the file manually it did the trick!

**Comment by mmahmoudian** [comment]
@boogiefromzk glad it helped. We are also working on a config validator and when that rolls out, the config file will be much better organized and you will get proper error if there is a mistake in the config.

**Comment by quicksketch** [comment]
@mmahmoudian Thanks all that info is awesome! I incorrectly assumed that `sudo apt install flameshot` would use the latest version (I'm on Ubuntu 20.04, the latest at this time), but it only installs `0.8.5-4`. Using the latest release from https://github.com/flameshot-org/flameshot/releases fixes everything. This is perfect. I don't care for adding a "ratio" feature. IMO it's best to keep the values separate, as is the new (current) behavior. Thank you so much!

**Comment by boogiefromzk** [comment]
Ratio was a 1-char-quick-fix suggestion, separate values are awesome!
